### ISSUE TITLE
Pages should open in a new tab when clicking on a logo

### ISSUE THREAD

**Issue by olejandro** [tokni/shift#38]
Currently pages open in the same page when clicking on the logos e.g., Tokni, EML, EMC.

**Comment by olavurellefsen** [comment]
I rather not :-/ See https://medium.com/the-metric/links-should-open-in-the-same-window-447da3ae59ba 

TLDR; The default behavior of hyperlinks is that they open within the same page.

**Comment by olavurellefsen** [comment]
https://github.com/tokni/shift/tree/olavur/ch1048/pages-should-open-in-a-new-tab-when-clicking

**Comment by olejandro** [comment]
I can definitely see the point for mobile devices. :-) Is it possible to
have different behaviour for desktop and for mobile?

**Comment by olavurellefsen** [comment]
Yes, it is possible to have different behaviour for desktop and mobile, but I still advise against it. It is not how the web generally works.

**Comment by olejandro** [comment]
Ok, let's keep it as it is then.
